tools/depends/target/dav1d/Makefile

```diff
@@ -0,0 +1,52 @@
+include ../../Makefile.include
+include DAV1D-VERSION
+DEPS = ../../Makefile.include Makefile
+
+# configuration settings
+CONFIGURE = $(NATIVEPREFIX)/bin/python3 $(NATIVEPREFIX)/bin/meson \
+	--buildtype=release \
+	--prefix=$(PREFIX) \
+	--libdir=lib \
+	--default-library=static \
+	-Denable_asm=true \
```

Comment:
probably going to have to hide this into a if check.

According to videolan's docs, they support asm for only certain things
https://code.videolan.org/videolan/dav1d#reached

Key take away, 32bit android and ios should be false i think for now. Most of our other platforms should be ok to leave it on.

Reply:
I'd rather leave it enabled for all. libdav1d's meson build will detect if it's possible to use asm or not. see, https://code.videolan.org/videolan/dav1d/blob/master/meson.build#L64-68